packages/cli/src/autorun/autorun.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * @license Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
```suggestion
 * @license Copyright 2019 Google Inc. All Rights Reserved.
```

don't need to bump this one :)